### PR TITLE
Test modified Mar 21

### PR DESCRIPTION
**Pull request by ghost**
Updated Scripts :
Test_TC_BOOL_1_1.yaml
Test_TC_BOOL_2_1.yaml
Test_TC_BRAC_1_1.yaml
Test_TC_ILL_1_1.yaml
Test_TC_LVL_1_1.yaml
Test_TC_LVL_3_1.yaml
Test_TC_LVL_4_1.yaml
Test_TC_LVL_5_1.yaml
Test_TC_LVL_6_1.yaml
Test_TC_MC_3_11.yaml
Test_TC_MC_3_7.yaml
Test_TC_MC_9_1.yaml
Test_TC_OCC_2_1.yaml
Test_TC_OCC_2_2.yaml
Test_TC_OO_2_1.yaml
Test_TC_PCC_1_1.yaml
Test_TC_PCC_2_1.yaml
Test_TC_PRS_2_1.yaml
Test_TC_PS_1_1.yaml
Test_TC_RH_1_1.yaml
Test_TC_RH_2_1.yaml
Test_TC_RH_2_2.yaml
Test_TC_TM_1_1.yaml
Test_TC_TM_2_2.yaml
Test_TC_TSTAT_1_1.yaml
Test_TC_TSTAT_2_1.yaml
Test_TC_TSUIC_1_1.yaml
Test_TC_TSUIC_2_1.yaml
Test_TC_TSUIC_2_2.yaml
Test_TC_WIFIDIAG_1_1.yaml

Attached exexution logs for reference:
[Execution_log.txt](https://github.com/project-chip/connectedhomeip/files/8316862/Execution_log.txt)

### CODE REVIEW

File: src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml [hunk 1]
```diff
@@ -40,3 +40,35 @@ tests:
...
+    - label: "Read the global attribute: EventList"
+      disabled: true
+      command: "readAttribute"
+      attribute: "EventList"
+      response:
+          constraints:
+              type: list
+
+    - label: "Read the global attribute: AcceptedCommandList"
+      disabled: true
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml:61]: This is supported, as of a few hours before this PR was created.

File: src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml [hunk 2]
```diff
@@ -40,3 +40,35 @@ tests:
...
+      command: "readAttribute"
+      attribute: "AcceptedCommandList"
+      response:
+          constraints:
+              type: list
+
+    - label: "Read the global attribute: GeneratedCommandList"
+      disabled: true
```
Comment (bzbarsky-apple) [inline src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml:69]: This is supported too.

File: src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml (hunk 1 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml:61]: i will rebase it and check

File: src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml (hunk 2 above)
Comment (manjunathambiger23) [reply src/app/tests/suites/certification/Test_TC_BRAC_1_1.yaml:69]: i will remove disabled if it is added while rebasing

**Review by jmeg-sfy — APPROVED** [review]

**Review by bzbarsky-apple — APPROVED** [review]

**Pull request merged**